### PR TITLE
Clean up string literals in code

### PR DESCRIPTION
**Pull request by lewischeng-ms**
Related issue: #151

### CODE REVIEW

File: src/Microsoft.Restier.Core/Conventions/ConventionalChangeSetConstants.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+        public const string AuthorizeMethodDataModificationUpdate = "Update";
+
+        public const string AuthorizeMethodDataModificationDelete = "Delete";
+
+        public const string AuthorizeMethodActionInvocationExecute = "Execute";
+#endregion
+
+#region Filtering method constants
+        public const string FilterMethodNamePrefix = "On";
```
Comment (karataliu) [inline src/Microsoft.Restier.Core/Conventions/ConventionalChangeSetConstants.cs]: consider remove this made other OnInsert OnUpdat, OnDelet? Also for the previous Can

File: src/Microsoft.Restier.Core/Microsoft.Restier.Core.csproj [hunk 2]
```diff
@@ -45,6 +45,7 @@
     <Compile Include="..\Shared\TypeExtensions.cs">
       <Link>Shared\TypeExtensions.cs</Link>
     </Compile>
+    <Compile Include="Conventions\ConventionalChangeSetConstants.cs" />
```
Comment (karataliu) [inline src/Microsoft.Restier.Core/Microsoft.Restier.Core.csproj]: better put to Line#50

File: src/Microsoft.Restier.EntityFramework/Submit/SubmitExecutor.cs [hunk 3]
```diff
@@ -40,7 +40,7 @@ private SubmitExecutor()
         public async Task<SubmitResult> ExecuteSubmitAsync(
             SubmitContext context, CancellationToken cancellationToken)
         {
-            DbContext dbContext = context.DomainContext.GetProperty<DbContext>("DbContext");
+            DbContext dbContext = context.DomainContext.GetProperty<DbContext>(DbDomainConstants.DbContextKey);
```
Comment (karataliu) [inline src/Microsoft.Restier.EntityFramework/Submit/SubmitExecutor.cs]: consider make this an internal method.

File: src/Microsoft.Restier.WebApi/ODataDomainController.cs [hunk 4]
```diff
@@ -41,6 +41,14 @@ namespace Microsoft.Restier.WebApi
     [ODataDomainExceptionFilter]
     public class ODataDomainController : ODataController
     {
+        private const string ETagGetterKey = "ETagGetter";
+        private const string ETagHeaderKey = "@etag";
+        private const string DefaultNameOfParameterExpression = "currentValue";
+        private const string PathTemplateForEntitySetWithKey = "~/entityset/key";
```
Comment (karataliu) [inline src/Microsoft.Restier.WebApi/ODataDomainController.cs]: i'll suggest not make template a const string.

File: src/Microsoft.Restier.Core/Conventions/ConventionalChangeSetConstants.cs (hunk 1 above)
Comment (lewischeng-ms) [inline src/Microsoft.Restier.Core/Conventions/ConventionalChangeSetConstants.cs]: Updated. Thanks.

File: src/Microsoft.Restier.Core/Microsoft.Restier.Core.csproj (hunk 2 above)
Comment (lewischeng-ms) [inline src/Microsoft.Restier.Core/Microsoft.Restier.Core.csproj]: Updated Thanks.

**Comment by karataliu** [comment]
Looks good.

File: src/Microsoft.Restier.EntityFramework/Submit/SubmitExecutor.cs (hunk 3 above)
Comment (lewischeng-ms) [inline src/Microsoft.Restier.EntityFramework/Submit/SubmitExecutor.cs]: Created issue #232

**Pull request merged**